### PR TITLE
Upgrade to latest and greatest Laravel 9

### PR DESCRIPTION
**Pull request by tcampbPPU**
```bash
Laravel Framework 9.2.0
```

### Replacements:
* `\Fruitcake\Cors\HandleCors::class` => `\Illuminate\Http\Middleware\HandleCors::class`
* `\Fideloper\Proxy\TrustProxies` => `\Illuminate\Http\Middleware\TrustProxies`

### Changes:
`app()->langPath()` now points to just `lang/` instead of `resources/lang` so moved that folder up a level

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -24,12 +24,15 @@
...
 Route::get('login', [Controller::class, 'auth'])->name('login');
-Route::get('redirect/{provider}', [AuthController::class, 'redirect'])->name('provider.redirect');
-Route::get('callback/{provider}', [AuthController::class, 'callback'])->name('provider.callback');
-Route::get('onetap/{credential}', [AuthController::class, 'onetap'])->name('onetap.support');
-Route::post('attempt', [AuthController::class, 'attempt'])->name('auth.attempt');
-Route::post('login', [AuthController::class, 'login'])->name('auth.login');
-Route::get('logout', [AuthController::class, 'logout'])->middleware('auth:api')->name('auth.logout');
-Route::get('me', [AuthController::class, 'me'])->middleware('auth:api')->name('auth.session');
+
+Route::controller(AuthController::class)->group(function () {
```
Comment (tcampbPPU) [inline routes/api.php:28]: Can now use Controller grouping as option

File: app/Http/Controllers/AuthController.php [hunk 2]
```diff
@@ -228,9 +228,13 @@ public function update(Request $request)
             ->option('name', 'required|string')
             ->option('avatar', 'required|url')
             ->verify();
-        auth()->user()->name ??= $request->name;
-        auth()->user()->avatar ??= $request->avatar;
-        auth()->user()?->save();
+
+        /** @var User $user */
```
Comment (tcampbPPU) [inline app/Http/Controllers/AuthController.php:232]: just cleaning up larastan issues

File: app/Http/Middleware/TrustProxies.php [hunk 3]
```diff
@@ -19,5 +19,10 @@ class TrustProxies extends Middleware
      *
      * @var int
      */
-    protected $headers = Request::HEADER_X_FORWARDED_ALL;
+    protected $headers =
```
Comment (tcampbPPU) [inline app/Http/Middleware/TrustProxies.php:22]: needed with new  `Illuminate\Http\Middleware\TrustProxies`

File: app/Http/Kernel.php [hunk 4]
```diff
@@ -16,7 +16,7 @@ class Kernel extends HttpKernel
     protected $middleware = [
         // \App\Http\Middleware\TrustHosts::class,
         \App\Http\Middleware\TrustProxies::class,
-        \Fruitcake\Cors\HandleCors::class,
+        \Illuminate\Http\Middleware\HandleCors::class,
```
Comment (tcampbPPU) [inline app/Http/Kernel.php:19]: Fruitcake\Cors\HandleCors is built into the framework now https://laravel-news.com/laravel-9-2-0

**Comment by acidjazz** [comment]
LGTM

**Pull request merged**